prodigy_demo/basic_recipe/text_classification_recipe.py

```diff
@@ -0,0 +1,99 @@
+"""
+Custom textcat recipe.
+
+To run this recipe:
+
+prodigy textcat_hf tweets_annotated \
+    ./data/data_to_label_500.jsonl \
+    -F text_classification_recipe.py
+    
+"""
+from typing import List, Optional, Iterator
+import copy
+
+import prodigy
+from prodigy.components.loaders import JSONL
+from prodigy.components.preprocess import add_tokens
+import spacy
+
+# here, we're loading a blank model
+nlp = spacy.blank("en")
+
+# now we're adding the huggingface model to the pipeline
+nlp.add_pipe(
+    "hf_text_pipe",
+    config={"model": "finiteautomata/bertweet-base-sentiment-analysis"},
+)
```

Comment:
sorry if this is obvious, but what exactly happens next? which of the models is it using below?

Reply:
it's using this model: https://huggingface.co/finiteautomata/bertweet-base-sentiment-analysis - will add link and clarify!